pkg/api/http_server.go

```diff
@@ -42,24 +42,31 @@ func init() {
 	})
 }
 
+type ProvisioningService interface {
+	ProvisionDatasources() error
+	ProvisionNotifications() error
+	ProvisionDashboards() error
+	GetDashboardFileReaderByName(name string) *dashboards.FileReader
```

Comment:
I think I would prefer to return the external Id instead of the full `dashboards.FileReader` Any reason for having the full reader returned to the Api?

Reply:
Yeah, will change, no reason to return the full reader here.

Reply:
Yeah I think there isn't a real reason for that, will change it.